src/services/areaService.ts

```diff
@@ -0,0 +1,29 @@
+import { IDistrict, IProvince, IWard } from '../interfaces/interface';
+import { axioInstance } from '../utils/request/httpRequest';
+
+export const findAllProvinces = async (): Promise<IProvince[]> => {
+  try {
+    const res = await axioInstance.get('provinces');
+    return res.data;
+  } catch (error) {
+    throw new Error();
+  }
+};
+
+export const findDistricts = async (id: number): Promise<IDistrict[]> => {
+  try {
+    const res = await axioInstance.get(`provinces/${id}`);
```

Comment:
```suggestion
    const res = await axioInstance.get<IDistrict[]>(`provinces/${id}`);
```

Viết type return cho generic của axios như thế này